antropy/entropy.py

```diff
@@ -550,19 +547,14 @@ def app_entropy(x, order=2, tolerance=0.2, metric="chebyshev", tolerance_referen
     -0.0010
     """
     # define r
-    if tolerance_reference =="SD":
-        r = tolerance * np.std(x, ddof=0)
-    elif tolerance_reference == "abs":
-        r = tolerance
+    if tolerance is None:
+        r = 0.2 * np.std(x, ddof=0)
     else:
-        raise ValueError(
-            "The given reference for the tolerance r of %s is not valid."
-            "The valid options are: SD and abs" %(tolerance_reference)
-        )
+        r = tolerance
```

Comment:
Can you please add `assert isinstance(tolerance, (float, int))` before this line?